tests/component/test_device_properties.py

```diff
@@ -0,0 +1,63 @@
+"""Tests for validating device properties for different basic data types."""
+import pytest
+
+from nidaqmx.constants import BusType, TriggerUsage
+
+
+@pytest.mark.parametrize("device_by_name", ["bridgeTester"], indirect=True)
+def test__device__bool_property__returns_value(device_by_name):
```

Comment:
FYI, "given device" does make sense, because that is an object that the test is given.

Reply:
Do you mean that the test name could be changed to `test__given_device__bool_property__return_value`?